Update locking.mdx (#31775)
hashicorp/terraform: website/docs/language/state/locking.mdx

State locking happens automatically on all operations that could write

> state. You won't see any message that it is happening. If state locking fails,
> Terraform will not continue. You can disable state locking for most commands

Reviewer: ```suggestion
state. You do not see any message that it happens. If state locking fails,
Terraform does not continue. You can disable state locking for most commands
```

Suggest changing "won't" and "will not" to "do not" and "does not" to avoid using contractions and to avoid future tense. Suggest changing "is happening" to "happens" to use fewer words.

> with the `-lock=false` flag but it is not recommended.

Reviewer: ```suggestion
with the `-lock=false` flag, but we do not recommend it.
```

Suggest changing "it is not recommended" to "we do not recommend it" to avoid a passive voice style issue.

> If acquiring the lock is taking longer than expected, Terraform will output
> a status message. If Terraform doesn't output a message, state locking is

Reviewer: ```suggestion
If acquiring the lock takes longer than expected, Terraform outputs
a status message. If Terraform does not output a message, state locking is
```

Suggest changing "is taking" to "takes" to use fewer words. Suggest changing "will output" to "outputs" to avoid future tense. Suggestion changing "doesn't" to "does not" to avoid using contractions.
